src/gluonts/model/renewal/_transform.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import numpy as np
+from gluonts.core.component import validated
+
+from gluonts.dataset.common import DataEntry
+from gluonts.dataset.field_names import FieldName
+from gluonts.transform import SimpleTransformation
+
+
+class AddAxisLength(SimpleTransformation):
+    """
+    Add the "length" of an array along the specified axis. For example,
```

Comment:
It's better to put a line break and leave a blank line after the first sentence ("For example..." in a new paragraph)